.github/workflows/delete-ns.yml

```diff
@@ -0,0 +1,36 @@
+name: delete-ns
+on:
+  workflow_dispatch:
+    inputs:
+      namespace:
+        type: choice
+        description: Which namespace you want to delete
```

Comment:
Description correction:
`Select the namespace you wish to delete"

Reply:
Done